sphinx-jsonschema/wide_format.py

```diff
@@ -211,10 +211,11 @@ def _dispatch(self, schema, label=None):
         rows = []
         self.nesting += 1
 
-        if 'definitions' in schema and self.options['lift_definitions']:
```

Comment:
Am I mistaken or did you drop the 'lift_definitions' option?
If you would please reintroduce it I will be pleased to merge your PR.

Reply:
Ha, good catch, should be fixed now.